spring-cloud-generator/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>4.1.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+    </parent>
+
+    <groupId>com.google.api</groupId>
```

Comment:
I kept this group ID same as the one used by gapic-generator-java jar, but not sure if it should be consistent with others from this repo (`com.google.cloud`) instead?

Reply:
I noticed that we have `java-core` in the `gapic-generator-java` monorepo [using](https://github.com/googleapis/gapic-generator-java/blob/main/java-core/pom.xml) the `com.google.cloud` group ID, unlike the `com.google.api` group as in other libraries in there, so an argument of consistency may not be as strong (although `java-core` was recently added to the monorepo).

IMO if we have no technical difficulties to keep the group ID consistent with `spring-cloud-gcp` then it may make more sense to assimilate it into spring-cloud-gcp rather than making it consistent with what is now a dependency of the spring generator.

Reply:
That's a good point! Yeah switching to use `com.google.cloud` here shouldn't be a technically difficult one, just wanted to check if we have any conventions/preferences here (in case we decide to also publish this generator jar in the future).

Reply:
I feel `com.google.cloud` makes more sense as I consider this project is closer to `Spring-cloud-gcp` than `gapic-generator-java`, as this project is very specific to `Spring-cloud-gcp`, but we could `gapic-generator-java` as a generic tool.